packages/renderer/src/lib/preferences/PreferencesRenderingItem.svelte

```diff
@@ -92,6 +92,16 @@ async function openGitHubDiscussion(): Promise<void> {
       <div class="flex flex-row text-[color:var(--pd-invert-content-card-text)]">
         <div class="flex flex-row space-x-2 items-center">
           <span class="font-semibold">{recordUI.title}</span>
+          {#if record.locked}
+            <Tooltip tip="This setting is managed by your organisation." right>
```

Comment:
nitpick: as the bot said in https://github.com/podman-desktop/podman-desktop/pull/14767/commits/c733b5e5c77d67d6cfa4999395ccc41c08f75630#r2497105830, the rest of the codebase uses `organization` with a 'z'.